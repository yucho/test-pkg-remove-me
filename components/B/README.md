# @test-pkg-remove-me/b

Welcome to my `B` component.

## Installation

```sh
npm i @test-pkg-remove-me/b
# or with yarn
yarn add @test-pkg-remove-me/b
```

Then to use the component in your code just import it!

```js
import B from '@test-pkg-remove-me/b';
// and with css-modules
import '@test-pkg-remove-me/b/dist/main.css';
```
