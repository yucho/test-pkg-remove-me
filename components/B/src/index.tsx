import React from 'react';
import styles from './B.css';

interface BProps {
  /** The value for the component to display */
  value?: string;
}

/** A simple component. */
export const B = (props: BProps) => {
  return (
    <>
      <div className={styles.b}>This is a B component {props.value}</div>
      <div>Add one line here.</div>
    </>
  );
};

export default B;
