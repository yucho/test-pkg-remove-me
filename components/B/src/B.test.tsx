import React from 'react';
import { render } from '@testing-library/react';
import B from '.';

describe('B', () => {
  test('It matches the snapshot', () => {
    const { container } = render(<B />);
    expect(container).toMatchSnapshot();
  });
});
