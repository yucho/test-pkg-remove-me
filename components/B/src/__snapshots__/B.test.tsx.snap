// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`B It matches the snapshot 1`] = `
<div>
  <div
    class="b"
  >
    This is a B component 
  </div>
</div>
`;
